CHANGELOG.md

```diff
@@ -160,7 +160,8 @@ For compatibility reasons with Symfony 5.2 and PHP 8, we do not test these legac
 ## 2.5.0 beta 1
 
 * Add an HTTP client dedicated to functional API testing (#2608)
-* Add PATCH support (#2895)
+* Add PATCH support (#2895)  
+  Note: with patch, responses will skip null values. As this may break on some endpoints, to enable patch support you need to manually [add the `merge-patch+json` format](https://api-platform.com/docs/core/content-negotiation/#configuring-patch-formats). This will be the default behavior in API Platform 3.
```

Comment:
```suggestion
  Note: with PATCH, responses will skip null values. As this may break on some endpoints, you need to manually [add the `merge-patch+json` format](https://api-platform.com/docs/core/content-negotiation/#configuring-patch-formats) to enable PATCH support. This will be the default behavior in API Platform 3.
```